src/core/services/theming/theming.js

```diff
@@ -75,7 +75,7 @@ var DARK_SHADOW = '1px 1px 0px rgba(0,0,0,0.4), -1px -1px 0px rgba(0,0,0,0.4)';
 var LIGHT_SHADOW = '';
 
 var DARK_CONTRAST_COLOR = colorToRgbaArray('rgba(0,0,0,0.87)');
```

Comment:
is there any reason why these arent just defined as arrays but instead we convert it each run?

eg.
```javascript
var DARK_CONTRAST_COLOR = [ 0, 0, 0, 0.87 ]
var LIGHT_CONTRAST_COLOR = [ 255, 255, 255, 0.87 ]
var STRONG_LIGHT_CONTRAST_COLOR = [ 255, 255, 255 ]
```